packages/api/src/external/carequality/command/cq-directory/cq-gateways.ts

```diff
@@ -14,12 +14,21 @@ export async function setEntriesAsGateway(oids: string[]): Promise<void> {
   );
 }
 
-export async function getOrganizationIds(excludeManagingOrgs: string[]): Promise<string[]> {
+/**
+ * Returns the ID of CQ directory entries that have the managing organization's name
+ * matching the provided ones.
+ *
+ * @param managingOrgNames - An array of managing organization names
+ * @returns IDs of the CQ directory entries
+ */
+export async function getOrganizationIdsByManagingOrgs(
+  managingOrgNames: string[]
+): Promise<string[]> {
   const entries = await CQDirectoryEntryModel.findAll({
     attributes: ["id"],
     where: {
       managingOrganization: {
-        [Op.notIn]: excludeManagingOrgs,
+        [Op.notIn]: managingOrgNames,
```

Comment:
Is this intended? you're getting organizations, whose `managingOrganization` is NOT IN the list of `managingOrgNames`. So you're going to end up excluding everything but the target orgs, no?

Reply:
See [this thread](https://metriport.slack.com/archives/C04DMKE9DME/p1715711377475049?thread_ts=1715704101.580579&cid=C04DMKE9DME).